client/wfnews-war/src/main/angular/src/app/components/active-wildfire-map/active-wildfire-map.component.ts

```diff
@@ -479,11 +499,11 @@ export class ActiveWildfireMapComponent implements OnInit, AfterViewInit {
     this.resizeBoxElement.style.height = `${window.innerHeight - this.lastPointerPosition + 20}px`
   }
 
-  openLink(link:string) {
+  openLink(link: string) {
     if (link === 'Disclaimer') {
       window.open('https://www2.gov.bc.ca/gov/content/home/disclaimer', "_blank");
     }
-    else if  (link === 'Privacy') {
+    else if (link === 'Privacy') {
       window.open('https://www2.gov.bc.ca/gov/content/home/privacy', "_blank");
```

Comment:
Not related to this PR, but those URLs should likely be added to the configuration rather than being hardcoded